decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
+
+### Discussion
+There must be minimum 2 weeks of discussion in the pull request thread.
```

Comment:
Some proposals are created from internal Magento stories or bugs with deadlines. Two weeks blocking a whole team or initiative is a long time. Are there exceptions to this rule? Maybe a more important question, are the guidelines clear for when a proposal MUST be made versus when a simple code review from an architect is sufficient?

Reply:
That's a really good point I would like to discuss. Maybe architecture team should initiate voting, not author.

Reply:
Architecture team should initiate voting as for me, as some proposals may be misleading or propose the idea for which we already have resolution